csrankings-c.csv

```diff
@@ -1318,6 +1318,7 @@ Concepción Vidal,University of A Coruña,https://pdi.udc.es/en/File/Pdi/TD3AF,G
 Concettina Guerra,Georgia Institute of Technology,http://www.cc.gatech.edu/~guerra,NOSCHOLARPAGE
 Cong Liu 0005,Univ. of California - Riverside,https://intra.ece.ucr.edu/~cong/index.html,vpc4bggAAAAJ
 Cong Shi 0004,NJIT,https://web.njit.edu/~cs638,5MvTNBEAAAAJ
+Cong Sun,Xidian University,https://web.xidian.edu.cn/suncong/index.html,GIHrcRMAAAAJ

```

Comment:
Needs to be disambiguated (I think it should be https://dblp.org/pid/45/5103-1.html).

Reply:
Modified Cong Sun professor's information to eliminate ambiguity.
Hope you review again. Thank you !